lib/Github/HttpClient/HttpClient.php

```diff
@@ -20,6 +20,11 @@
  */
 class HttpClient implements HttpClientInterface
 {
+    /**
+     * @var ClientInterface
+     */
+    public $client;
```

Comment:
Making it private would be breaking anyway since it was defaulting to public already?